packages/axway-cli-pm/src/utils.js

```diff
@@ -49,7 +49,6 @@ export class ListrTextRenderer {
 export async function runListr({ console, json, tasks }) {
```

Comment:
`console` is no longer used and can be removed from the args and jsdoc as well as remove `console` from being passed in from `purge.js`, `update.js`, `install.js`, and `uninstall.js`.

Reply:
Sure. I'll go remove it now.